src/status_im/subs.cljs

```diff
@@ -2268,13 +2289,35 @@
  (fn [multiaccount]
    (get multiaccount :profile-pictures-visibility)))
 
+(defn- replace-contact-image-uri [contact port identity]
+  (let [theme (colors/current-theme)
+        identicon (image-server/get-identicons-uri port identity theme)
+        contact-images (:images contact)
+        contact-images (reduce (fn [acc image] (let [image-name (:type image)
+                                                       ; We pass the clock so that we reload the image if the image is updated
```

Comment:
This is some form of hack? 🤔 if so, is there no cleaner way of reloading the image on updates?

Reply:
hi @J-Son89  the "cleaner" way would be generate the image link on the status-go side, but there is some special params like "current theme / new-ui-enabled?" status-go don't know , if u have better idea, i'd like to hear :)